content/addons.md

```diff
@@ -104,9 +112,9 @@ However, in the context of an addon, this creates more files than we would see i
 
 ```
 
-Some files go in the `app` directory, while others go into the `addon` directory. We'll start by looking at the addon directory. At this stage, whatever we put in the `<addon-name>.hbs` file is what could be used immediately in an app.
+Some files go in the `app` directory, while others go into the `addon` directory. We'll start by looking at the addon directory. Whatever we put in the `<addon-name>.hbs` file is what could be used immediately in an app, and will be referenced in templates as `{{addon-name}}`. Later on in this guide, we will cover how to export multiple templates with different names from a single addon project.
```

Comment:
This part is hard for me to follow, and I'm not convinced it's correct, but even if it is this pattern is not a good thing to teach in a beginners guide. This whole single-templates special case is not a real thing anybody uses.

The canonical, well-supported, idiomatic way to make a component in an addon is to run `ember generate component your-component` and use the files it generates:

 - `/app/components/your-component.js` which reexports from
 - `/addon/components/your-component.js` which imports its template explicitly from
 - `/addon/templates/components/your-component.hbs`